test/pummel/test-stream-pipe-multi.js

```diff
@@ -49,10 +49,6 @@ process.on('exit', function() {
   assert.equal(cnt, rclosed, 'readable streams closed');
 });
 
-for (var i = 0; i < chunkSize; i++) {
-  chunkSize[i] = i % 256;
-}
-
```

Comment:
I *think* it's safe to remove because I don't think the contents of the buffer affects anything in the test. But regardless, I can see how failing to initialize the buffer contents doesn't pass the smell test even if those contents are unused. So I'll put some initialization back.

It's odd because the modulus operator is superfluous because the buffer has a length of 250 so `% 256` doesn't do anything.

And all this stuff was in the very first commit for the test. So it seems likely that this test has never run successfully.